### PR TITLE
Add dark mode

### PR DESCRIPTION
**Pull request by aurorasmiles**
Implements #50 
Preview: https://aurora.cutiepie.at/papermc.io/dist/community

Any feedback would be greatly appreciated. For the colors I oriented myself at Dark Reader.

### CODE REVIEW

File: src/css/styles.scss [hunk 1]
```diff
@@ -3,6 +3,39 @@ body {
...
+  .sidenav {
+    background-color: #181a1b;		
+  }
+  
+  .sidenav .grey-text.text-darken-4 {
+    color: #e8e6e3 !important
+  } 
+  
+  .sidenav li>a>i.material-icons {
+	color: #e8e6e3;
```
Comment (KennyTV) [inline src/css/styles.scss:34]: _cries in tabs_

**Comment by DemonWav** [comment]
Can you fix the files which don't have a line ending at the end of the file?

**Review by DemonWav — APPROVED** [review]
Looks great.

**Comment by aurorasmiles** [comment]
> 
> 
> Can you fix the files which don't have a newline at the end of the file?

done

**Pull request merged**